### PR TITLE
set curl to follow redirects to fix scraper error

### PR DESCRIPTION
**Pull request by jrassa**
Fix for scraper issue reported [here](https://retropie.org.uk/forum/topic/17925/emulationstation-scraping-thegamesdb-net-api-change)

### CODE REVIEW

File: es-core/src/HttpReq.cpp [hunk 1]
```diff
@@ -58,6 +58,15 @@ HttpReq::HttpReq(const std::string& url)
...
 	}
 
+	//set curl to handle redirects
+	err = curl_easy_setopt(mHandle, CURLOPT_FOLLOWLOCATION, 1L);
+        if(err != CURLE_OK)
+        {
+                mStatus = REQ_IO_ERROR;
+                onError(curl_easy_strerror(err));
+                return;
+        }
```
Comment (Koerty) [inline es-core/src/HttpReq.cpp]: Starting from the "if", you used white spaces instead of tabs.

**Comment by Thunder07** [comment]
Hi there, I'm currently the one working on the new revision of TheGamesDB website, just want to start with an apology as I failed to update this url in the api when the redirect was enabled, and would like to let you know that, that has been since fixed, but while you're at it you should change the api call domain location to the legacy subdomain as well as that main domain api will be turned off in time(yet to be decided but an announcement will be proceeded before that), but the legacy api will remain for a period to allow apps to start using the new json api (beta is out for testing, but I wouldn't recommend it for production yet)

**Comment by jrassa** [comment]
@Thunder07  Thanks for the quick response getting things fixed.  I will update this PR to use the legacy subdomain as recommended.

**Comment by jrassa** [comment]
I have amended this to use the new legacy url as suggested above by Zer0xFF.  If no one has any issues with this I will merge it tomorrow.

**Pull request merged**